shell/ci/testing/coverbot/upload-coverage.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Uploads code coverage file to S3 via getoutreach/coverbot
+# Note: This is not meant to be called outside of coverage.sh
+set -e
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+SHELL_DIR="$DIR/../../.."
+LIB_DIR="$SHELL_DIR/lib"
+
+# shellcheck source=../../../lib/bootstrap.sh
+source "${LIB_DIR}/bootstrap.sh"
+
+coverage_file="$1"
```

Comment:
If empty, exit with an error? Also, consider validating that arguments is only equal to 1.

Reply:
Oops! Could have sworn I had a check for an empty file at one point, not sure what happened to it...